Tools/ardupilotwaf/chibios.py

```diff
@@ -492,6 +492,8 @@ def load_env_vars(env):
         else:
             env[k] = v
             print("env set %s=%s" % (k, v))
+    if env.DEBUG:
```

Comment:
needs to cope with -g configure 
maybe alias -g with --debug-symbols, then check for env.DEBUG or env.DEBUG_SYMBOLS

Reply:
Done.